src/client/apps/edit/components/admin/components/article/article_publish_date.tsx

```diff
@@ -1,17 +1,19 @@
-import colors from "@artsy/reaction/dist/Assets/Colors"
+import { Button, color, Flex } from "@artsy/palette"
 import moment from "moment"
-import styled from "styled-components"
-import PropTypes from "prop-types"
 import React, { Component } from "react"
 import ReactDOM from "react-dom"
-import { avantgarde } from "@artsy/reaction/dist/Assets/Fonts"
-import { ButtonMedium } from './button_medium'
+import { ArticleData } from "reaction/Components/Publishing/Typings"
+import styled from "styled-components"
+import { FormLabel } from "../../../../../../components/form_label"
 
-export class ArticlePublishDate extends Component {
-  static propTypes = {
-    article: PropTypes.object,
-    onChange: PropTypes.func,
-  }
+interface ArticlePublishDateProps {
+  article: ArticleData
+  onChange: (key: string, value: any) => void
+}
+
+export class ArticlePublishDate extends Component<ArticlePublishDateProps> {
+  public date
+  public time
```

Comment:
Are there any objects that actually reference one of these members? Otherwise they could be private. Also specifying a type would be helpful since the names `date` and `time` imply they are plain value objects:

```ts
private date: HTMLInputElement
private time: HTMLInputElement
```

Reply:
This is for testing

Reply:
React components obscure these props by default, so no need to add `private` (or `public`). These public / private designators do work well for normal classes however.

Reply:
Is there an example of a ref that doesn't use these I can look at?  I made them public so I could alter the values in tests.